UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/ControlAction.cs

```diff
@@ -12,13 +12,40 @@ public class ControlAction : MonoBehaviour
 
 		public Image pullImage;
 
+		public static bool ThrowHold = true;
+
 		private void Start()
 		{
 			UIManager.IsThrow = false;
 
 			pullImage.enabled = false;
+			ThrowHold = GetThrowPreference();
+		}
+
+		public static bool GetThrowPreference()
+		{
+			return 1 == PlayerPrefs.GetInt("ThrowHoldPreference", 1);
```

Comment:
ever since alpha launch, the other behaviour has been the default one. Are you sure you want to make hold the deault now?

Reply:
yeah,  anyone can change it back in the preferences if they would like, as I think it's more user-friendly to have it a hold thing,  since a new user could press it accidentally and then wonder why they throw when they click